### PR TITLE
feat(League of Angels: Pact): add presence

### PR DESCRIPTION
**Pull request by CH40S734D3R**
## Description 

League of Angels: Pact is the latest action-idle MMORPG in the League of Angels series. Participate in fierce battles, awaken hundreds of Angels and empower them and finally, defend the Holy Land of Justice with their help!
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Screenshot 2024-06-21 191523

Screenshot 2024-06-21 191554
</details>

### CODE REVIEW

File: websites/L/LOAP/metadata.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "1214632755158319206",
+		"name": "arkhanths"
+	},
+	"service": "LOAP",
+	"description": {
+		"en": "Unofficial Discord Rich Presence for League of Angels: Pact. Works with R2Games and Game Hollywood."
+	},
```
Comment (Bas950) [inline websites/L/LOAP/metadata.json:10]: This should be a description of what LOAP is, not the Presence.
If possible, get an official description of a website

File: websites/L/LOAP/metadata.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+	"url": [
+		"nsh5api.gamehollywood.com",
+		"nsh5apisy.gamehollywood.com",
+		"r2games.com",
+		"www.r2games.com",
+		"gamehollywood.com",
+		"www.gamehollywood.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://r2.arkhanths.com/logo.jpg",
```
Comment (Bas950) [inline websites/L/LOAP/metadata.json:20]: Logo must be 512x512

File: websites/L/LOAP/presence.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+const presence = new Presence({ clientId: "1252257462476738580" }),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://r2.arkhanths.com/logo.jpg",
```
Comment (Bas950) [inline websites/L/LOAP/presence.ts:6]: Ditto for largeImagekey

File: websites/L/LOAP/presence.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+			startTimestamp: browsingTimestamp,
+		},
+		path = location.pathname + location.search,
+		showUID: boolean = await presence.getSetting<boolean>("uid");
+
+	if (path.includes("/web/oz/login.php")) {
+		const uid = new URL(
+			window.location.origin + window.location.pathname + window.location.search
```
Comment (Bas950) [inline websites/L/LOAP/presence.ts:14]:
```suggestion
			document.location.href
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Maybe instead of setting the activity in every if case, set it once at the end of all if cases.

File: websites/L/LOAP/metadata.json [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+		"www.gamehollywood.com"
+	],
+	"version": "1.0.0",
+	"logo": "https://r2.arkhanths.com/logo_512.jpg",
+	"thumbnail": "https://r2.arkhanths.com/thumbnail.jpg",
+	"color": "#2BA2CE",
+	"category": "games",
+	"tags": [
+		"loap"
+	],
```
Comment (Bas950) [inline websites/L/LOAP/metadata.json:26]: Maybe add some more tags, like `mmorpg`, etc.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Maybe add `League of Angels: Pact` as an alt name (`"altnames": ["League of Angels: Pact"],`) so users can search for this in the store too. Or just rename the whole presence to it.

**Review by Bas950 — APPROVED** [review]

**Review by Timeraa — APPROVED** [review]

**Pull request merged**